entity-framework/toc.yml

```diff
@@ -233,6 +233,9 @@
           href: core/providers/sql-server/indexes.md
         - name: Memory-optimized tables
           href: core/providers/sql-server/memory-optimized-tables.md
+        - name: Spatial data
+          displayName: GIS
```

Comment:
Should we leave spatial data as the display name? Am just not sure everyone knows the name GIS. Not sure.

Reply:
We should add Spatial Data too for localization.

Reply:
lol, `displayName` has nothing to do with the displayed name. (But I can certainly see why you might think that.) It for additional keywords users can search for:
![image](https://user-images.githubusercontent.com/1226749/94970333-4a7b7d80-04b9-11eb-8202-5d23db353b7e.png)